content/docs/reference/php-reference.md

```diff
@@ -9,49 +9,147 @@ menu:
+The PHP buildpack supports the full [software bill of materials]({{< ref
+"docs/concepts/sbom" >}}) (SBOM) in [Syft][format/syft],
+[CycloneDX][format/cyclonedx], [SPDX][format/spdx], and
+[Paketo-specific][format/paketo] formats.
 
-### MIBDIRS
+Apps built with the buildpack that use Composer as a a package manager contain
+SBOM entries that provide a full picture of the packages on the final app image
+that get installed.  Additionally, there are entries for the PHP version used,
+the Composer version used. Check out the [Access the Software Bill of Materials
+guide]({{< ref "docs/howto/sbom" >}}) for more information about how to
+retrieve the SBOM for your app image.
 
-* Set by: `php-dist` buildpack
-* Phases: `build` and `launch`
-* Value: See [php documentation](https://www.php.net/manual/en/snmp.installation.php)
+
+## Buildpack Configurations
+
+The buildpacks in the PHP language family set default configurations, as well
+as include configurations from user-provided sources. The sections below
+outline the various configurations the buildpacks provide during the build process, and
+the various locations that will also be considered as configuration sources.
+
+### [PHP Distribution Buildpack][bp/php-dist] Configuration
+1. The [default PHP `ini` file][bp/php-dist-ini] found from the PHP
+   Distribution itself, with no modifications. It can be found on the container
+   inside the PHP Distribution Buildpack layer under `/etc/php.ini`.
+2. [Buildpack-specific configuration][bp/php-cnb-ini] which can be found inside the PHP
+   Distribution Buildpack layer under `/etc/buildpack.ini`. This includes
+   confgurations specific to the buildpack's install process and app image file
+   system, such as the path to PHP extensions (`extension_dir`), the include
+   path to the PHP library directory.
+3. User-provided configurations from the `<APP-ROOT>/.php.ini.d` directory.
+
+Check out the [PHP_INI_SCAN_DIR documentation section]({{< ref
+"docs/reference/php-reference#php_ini_scan_dir" >}}) for more information.
+
+### [PHP FPM Buildpack][bp/fpm] Configuration
+The configuration from the PHP FPM buildpack is considered in order of
+precedence, from lowest to highest, with the highest precedence configuration
+sources overriding conflicting settings from lower precedence sources.
+
+1. The [default PHP FPM setting][bp/php-dist-fpm] found from the PHP
+   Distribution itself under `$PHPRC/php-fpm.d/www.conf.default`.
+2. [Buildpack-specific configuration][bp/php-cnb-fpm] which includes
+   configurations specific to the buildpack's role in the build process. This
+   includes settings such as the default listen directive, and other features
+   to make FPM work well with web servers.
+3. Configuration set by other buildpacks (such as the PHP Nginx
+   Buildpack [FPM configuration][bp/nginx-fpm]), which can be located at
+   `/workspace/.php.fpm.bp/*.conf`.
+4. User-provided configurations located from `<APP-ROOT>/.php.fpm.d/*.conf`.
```

Comment:
```suggestion
Configuration for PHP FPM comes from a few sources, listed here in order of precedence from highest to lowest:

1. User-provided configurations located in `<APP-ROOT>/.php.fpm.d/*.conf`.
2. Configuration set by other buildpacks (such as the PHP Nginx
   Buildpack [FPM configuration][bp/nginx-fpm]), which is stored in 
   `/workspace/.php.fpm.bp/*.conf` files.
3. [Buildpack-specific configuration][bp/php-cnb-fpm], which includes
   configurations specific to the buildpack's role in the build process. This
   includes settings such as the default `listen` directive, and other features
   to make FPM work well with web servers.
4. The [default PHP FPM configuration][bp/php-dist-fpm] that is distributed alongside the PHP
   Distribution itself. This is stored at `$PHPRC/php-fpm.d/www.conf.default` in the container.

```
Also, where does the configuration from the PHP-FPM buildpack live?

Reply:
It's linked under default PHP FPM setting